terraform-modules/aws/opensearch/main.tf

```diff
@@ -0,0 +1,139 @@
+resource "aws_cloudwatch_log_group" "opensearch_slow_logs" {
+  name              = "opensearch-slow-logs"
```

Comment:
THe name should have a prefix variable in it.  What would happen if you wanted to launch two of these in the same aws account?  I would imagine it would error out saying this log group name already exist.